### ISSUE TITLE
How to unload swf to release memory?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#66]
```
What steps will reproduce the problem?
1.flexPaperViewer.SwfFile = "http:localhost\download\teste1.swf"
2.flexPaperViewer.SwfFile = "http:localhost\download\teste2.swf"
3.

What is the expected output? What do you see instead?
-memory of previous swf and not released

What version of the product are you using? On what operating system?
-Internete explorer 8, 
-flash player 10
-1.2.4 Flex Version 

Please provide any additional information below.

If you try to load the document several times, the browser crashes after 
some the time the memory is not garbage collected Being.
I'm reading via url dynamically.
How to unload swf to release memory.

```

Original issue reported on code.google.com by `wpso...@gmail.com` on 11 May 2010 at 1:17

**Comment by GoogleCodeExporter** [comment]
```
I would really interested to know the best process to load multiple document 
within 
the same instance.
```

Original comment by `namdeo.p...@gmail.com` on 17 May 2010 at 11:55

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@gmail.com` on 18 May 2010 at 3:34
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.3 final
```

Original comment by `erik.eng...@gmail.com` on 21 Jun 2010 at 9:34
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
In the new version (1.3) I had to comment line 269 of Viewer.as to read my swf.
Read my swf dynamically via a URL.

/ * If (_libMC == null) (if (_libMC.parent! = Null) (_libMC.parent.removeChild 
(_libMC);) _libMC.unload ();) * / _libMC = null;
```

Original comment by `wpso...@gmail.com` on 21 Jun 2010 at 1:02
